static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -299,6 +273,4 @@ const TooltipWrapper = styled('span')`
   padding-left: ${space(0.5)};
 `;
 
-const GroupSidebar = withApi(withSentryRouter(BaseGroupSidebar));
-
-export default GroupSidebar;
+export default withApi(withSentryRouter(GroupSidebar));
```

Comment:
Please give it a name, otherwise typescript doesn't know the name of this anonymous function.

You can `useApi` here as well as `useRouter` probably as well, so then you won't even need to wrap it.